scRNA-seq-advanced/02-celltype_assignment.Rmd

```diff
@@ -18,13 +18,18 @@ This notebook will demonstrate how to:
 
 In this notebook, we will attempt to assign cell types to each of the cells in a dataset, using some of the automated tools that are available within the Bioconductor universe.
 
-
 Much of the material in this notebook is directly inspired by, and draws heavily on, material presented in the book [_Orchestrating Single Cell Analysis with Bioconductor_](http://bioconductor.org/books/3.16/OSCA/). 
 
-The data we will use for this notebook is derived from a 10x Genomics Dataset: https://software.10xgenomics.com/single-cell-gene-expression/datasets/6.0.0/10k_PBMCs_TotalSeq_B_3p. 
-The data here have already been filtered, normalized, and had dimension reductions calculated.
+The data we will use for this notebook is derived from a [10x Genomics dataset of human peripheral blood mononuclear cells (PBMCs)](https://software.10xgenomics.com/single-cell-gene-expression/datasets/6.0.0/10k_PBMCs_TotalSeq_B_3p.)
+These data include both single cell RNA-seq counts and quantification of antibody-derived tags (ADTs) performed by sequencing short DNA barcodes attached to specific antibodies. 
+This type of ADT sequencing with single cells is commonly known as CITE-seq, after the protocol developed by [Stoeckius _et al._ (2017)](https://doi.org/10.1038/nmeth.4380).  
+The antibodies used here are the [The TotalSeq™-B Human TBNK Cocktail](https://www.biolegend.com/en-us/products/totalseq-b-human-tbnk-cocktail-19043), a set of antibodies designed to react with immune cell surface markers.
+
+The data here have already been filtered, normalized, and had dimension reductions calculated for the single-cell RNA-seq data.
+The ADT data has also been separately filtered and normalized.
+For details about how to perform these tasks with data that has been processed with Cell Ranger, have a look at our `exercise_01` notebook in this directory.
```

Comment:
Noting this notebook links the relevant OSCA chapter too so that's set!
Eventually fill the nb name all the way in, once the name is definitely set.